lib/kramdown_rpf/rpf.rb

```diff
@@ -172,6 +202,79 @@ def self.convert_task_to_html(task)
         HEREDOC
       end
 
+      module KnowledgeQuiz
+        def self.convert_feedback_to_html(feedback, index = nil)
+          id = 'feedback'
+          id += "-for-choice-#{index + 1}" unless index.nil?
+          <<~HEREDOC
+            <li class="knowledge_quiz__feedback-item" id="#{id}">
+            #{::Kramdown::Document.new(feedback.strip, KRAMDOWN_OPTIONS).to_html.strip}
+            </li>
+          HEREDOC
+        end
+  
+        def self.convert_label_to_html(label, index, checked)
+          number = index + 1
+          <<~HEREDOC
+            <label for="choice-#{number}">#{::Kramdown::Document.new(label, KRAMDOWN_OPTIONS).to_html.strip}</label>
+            <input type="radio" name="answer" value="#{number}" id="choice-#{number}" #{checked ? 'checked': ''}/>
+          HEREDOC
+        end
+  
+        def self.convert_choices_to_html(text)
```

Comment:
This is quite a chunky method, tried to figure ways to break down without passing objects around too much or allowing called methods to mutate things, but not much luck.
Also difficult to have a single regex to check for feedback and split a 'choice' into it's constituent parts.  Feel that this regex would possibly be quite complicated.